src/app/zwiazek/kalendarz/page.tsx

```diff
@@ -1,5 +1,373 @@
-import { ComingSoon } from "@/components/ComingSoon/ComingSoon";
+"use client";
+import { Breadcrumbs } from "@/components/shared/Breadcrumbs/Breadcrumbs";
+import { Heading } from "@/components/shared/Heading/Heading";
+import { useState } from "react";
+import { Button } from "@/components/shared/Button/Button";
+import { CalendarCard } from "@/components/shared/CalendarCard/CalendarCard";
+import Container from "@/components/shared/Container";
+import IconArrowCircle from "@/icons/IconArrowCircle";
+import clsx from "clsx";
+import Image from "next/image";
 
 export default function CalendarPage() {
-  return <ComingSoon />;
+  const daysOfWeek = ["Pon", "Wt", "Śr", "Czw", "Pt", "Sob", "Ndz"];
+  const months = [
+    "Styczeń",
+    "Luty",
+    "Marzec",
+    "Kwiecień",
+    "Maj",
+    "Czerwiec",
+    "Lipiec",
+    "Sierpień",
+    "Wrzesień",
+    "Październik",
+    "Listopad",
+    "Grudzień",
+  ];
+
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const today = new Date();
+
+  const handleNextMonth = () => {
+    setCurrentDate(new Date(currentDate.setMonth(currentDate.getMonth() + 1)));
+  };
+  const handlePrevMonth = () => {
+    setCurrentDate(new Date(currentDate.setMonth(currentDate.getMonth() - 1)));
+  };
+
+  const generateCalendar = () => {
```

Comment:
This function could be static and outside of the component, accepting the needed data via arguments.